src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/17.diff.sql

```diff
@@ -0,0 +1,31 @@
+﻿/****************************************************************************************
+Guidelines to create migration scripts - https://github.com/microsoft/healthcare-shared-components/tree/master/src/Microsoft.Health.SqlServer/SqlSchemaScriptsGuidelines.md
+This diff is broken up into several sections:
+ - The first transaction contains changes to tables and stored procedures.
+ - The second transaction contains updates to indexes.
+ - IMPORTANT: Avoid rebuiling indexes inside the transaction, it locks the table during the transaction.
+******************************************************************************************/
+
+SET XACT_ABORT ON
+
+BEGIN TRANSACTION
+GO
+
+-- Data Type INT = 56
+IF EXISTS (SELECT * FROM sys.sequences WHERE Name = 'WorkitemKeySequence' AND system_type_id = 56)
```

Comment:
Did you had a chance to execute? I get "The COMMIT TRANSACTION request has no corresponding BEGIN TRANSACTION." error

Reply:
I ran it through in my local. and also through the schema upgrade tests. I did not run into the transaction error. Will nuke my database and try it again.